csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -1,3 +1,146 @@
+
+/**
+ * An invocation of the compiler. Note that more than one file may be
+ * compiled per invocation. For example, this command compiles three
+ * source files:
+ *
+ *   gcc -c f1.c f2.c f3.c
```

Comment:
I realize that this was copied from C++, but should we have a `csc` example instead?

Reply:
This is a general thing for all the comments, they are very C++ specific.